master/docs/manual/cfg-workers-openstack.rst

```diff
@@ -106,12 +106,14 @@ These are the same details set in either environment variables or passed as opti
 ``nova_args``
     (optional)
     A dict that will be appended to the arguments when creating a VM.
-    Buildbot uses the OpenStack Nova version 1.1 API by default (see client_version).
+    Buildbot uses the OpenStack Nova version 2 API by default (see client_version).
```

Comment:
I would add this answer in the documentation(and release note). Not sure how deprecated is 1.1